frontend/src/app/shared/components/code/code.component.html

```diff
@@ -3,17 +3,15 @@
         <h5 class="name">
             Solution
         </h5>
-        <img src="assets/full-size.svg" alt="full-size">
```

Comment:
According to design we have resize btn, but question is whether we need to add two sections for the code in one screen

Reply:
But according to task - we have only one section and don't have resize button: [task](https://trello.com/c/LhLLhxmS/119-as-a-user-i-want-to-have-code-component)